streamer/autodetect.py

```diff
@@ -119,9 +119,10 @@ def get_frame_rate(input: Input) -> Optional[float]:
     return None
 
   # This string is the framerate in the form of a fraction, such as '24/1' or
-  # '30000/1001'.  We must split it into pieces and do the division to get a
+  # '30000/1001'.  Occasionally, there is a pipe after the framerate, such as
+  # '32700/1091|'.  We must split it into pieces and do the division to get a
   # float.
-  fraction = frame_rate_string.split('/')
+  fraction = frame_rate_string.split('|')[0].split('/')
```

Comment:
@joeyparrish, sorry for the late comment, but I observed the same issue for the [resolution_string](https://github.com/shaka-project/shaka-streamer/pull/128/files#diff-384852858464e27c844053b0588796a8a22150b5c2a7d02d6dda1383df915e92R151) too.

```
Traceback (most recent call last):
  File "/usr/local/bin/shaka-streamer", line 126, in <module>
    sys.exit(main())
  File "/usr/local/bin/shaka-streamer", line 105, in main
    with controller.start(args.output, input_config_dict, pipeline_config_dict,
  File "/Users/hmishinev/Projects/Reedsy/shaka-streamer/streamer/controller_node.py", line 179, in start
    self._input_config = InputConfig(input_config_dict)
  File "/Users/hmishinev/Projects/Reedsy/shaka-streamer/streamer/input_configuration.py", line 405, in __init__
    super().__init__(dictionary)
  File "/Users/hmishinev/Projects/Reedsy/shaka-streamer/streamer/configuration.py", line 325, in __init__
    value = self._check_and_convert_type(field, key, value)
  File "/Users/hmishinev/Projects/Reedsy/shaka-streamer/streamer/configuration.py", line 381, in _check_and_convert_type
    return [self._check_and_convert_type(subfield, key, v) for v in value]
  File "/Users/hmishinev/Projects/Reedsy/shaka-streamer/streamer/configuration.py", line 381, in <listcomp>
    return [self._check_and_convert_type(subfield, key, v) for v in value]
  File "/Users/hmishinev/Projects/Reedsy/shaka-streamer/streamer/configuration.py", line 368, in _check_and_convert_type
    sub_object = field.type(value)
  File "/Users/hmishinev/Projects/Reedsy/shaka-streamer/streamer/input_configuration.py", line 248, in __init__
    self.resolution = autodetect.get_resolution(self)
  File "/Users/hmishinev/Projects/Reedsy/shaka-streamer/streamer/autodetect.py", line 156, in get_resolution
    width_string, height_string = resolution_string.split('|')
ValueError: too many values to unpack (expected 2)
```

Reply:
Since I don't have the content that is triggering this, can you please fork and send a PR that resolves all autodetect issues for you?

Reply:
Sure thing 👍

Reply:
@joeyparrish, https://github.com/shaka-project/shaka-streamer/pull/129 addresses all the issues that I met.

Reply:
Thank you!